webpack.config.js

```diff
@@ -19,63 +19,49 @@
 const _ = require('lodash')
 const path = require('path')
 const SimpleProgressWebpackPlugin = require('simple-progress-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 const nodeExternals = require('webpack-node-externals')
 
 const commonConfig = {
   mode: 'production',
-  optimization: {
-    // Minification breaks angular.
-    minimize: false
-  },
-  target: 'electron-main',
-  module: {
-    rules: [
-      {
-        test: /\.jsx?$/,
-        include: [ path.resolve(__dirname, 'lib/gui') ],
-        use: {
-          loader: 'babel-loader',
-          options: {
-            presets: [
-              '@babel/preset-react',
-              [ '@babel/preset-env', { targets: { electron: '3' } } ]
-            ],
-            plugins: [ '@babel/plugin-proposal-function-bind' ],
-            cacheDirectory: true
-          }
-        }
-      },
-      {
-        test: /\.html$/,
-        include: [ path.resolve(__dirname, 'lib/gui/app') ],
-        use: {
-          loader: 'html-loader'
-        }
-      }
-    ]
-  },
   resolve: {
     extensions: [ '.js', '.jsx', '.json' ]
   },
   plugins: [
     new SimpleProgressWebpackPlugin({
       format: process.env.WEBPACK_PROGRESS || 'verbose'
     })
-  ]
+  ],
+  output: {
+    path: path.join(__dirname, 'generated'),
+    filename: '[name].js'
+  }
 }
 
-const guiConfig = _.assign({
+const guiConfig = _.assign(_.cloneDeep(commonConfig), {
```

Comment:
You can use `_.assign({}, commonConfig, { other: stuff })` and avoid the `cloneDeep`.